lnwallet/commitment.go

```diff
@@ -177,6 +185,21 @@ func DeriveCommitmentKeys(commitPoint *btcec.PublicKey,
 		)
 	}
 
+	// If this is a taproot commitment, then we'll use the funding keys to
+	// generate a combined funding key.
+	if chanType.IsTaproot() {
+		musigKey, _, _, _ := musig2.AggregateKeys(
+			[]*btcec.PublicKey{
+				localChanCfg.MultiSigKey.PubKey,
+				remoteChanCfg.MultiSigKey.PubKey,
+			},
+			true,
+			musig2.WithBIP86KeyTweak(),
```

Comment:
Not sure if I follow, the tweak is needed to make the proper funding key we put in the end script. 

Revisiting this area though, we no longer need `CombinedFundingKey` at all, since we use the NUMS key here now in place, will remove from the commits.